common/src/main/java/com/wynntils/gui/screens/WynntilsQuestBookScreen.java

```diff
@@ -207,6 +223,22 @@ public void onQuestsReloaded(QuestBookReloadedEvent.QuestsReloaded event) {
         }
     }
 
+    @SubscribeEvent
+    public void onQuestsReloaded(QuestBookReloadedEvent.MiniQuestsReloaded event) {
+        if (!miniQuestMode) return;
+
+        this.setQuests(QuestBookManager.getMiniQuests());
+
+        for (QuestInfo quest : quests) {
+            if (!quest.isTracked()) {
```

Comment:
Can you track mini-quests? I did not know that. I've only read the signs and then come back and clicked with the material...

Reply:
You can. I added the ability to track them in the quests manager so I know for sure it is possible.